Sprint-1/interpret/to-pounds.js

```diff
@@ -3,19 +3,21 @@ const penceString = "399p";
 const penceStringWithoutTrailingP = penceString.substring(
   0,
   penceString.length - 1
-);
+);//delete a p 
 
-const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0");
+const paddedPenceNumberString = penceStringWithoutTrailingP.padStart(3, "0"); //make sure three charachter.
 const pounds = paddedPenceNumberString.substring(
   0,
   paddedPenceNumberString.length - 2
-);
+);// compare to pound.
 
 const pence = paddedPenceNumberString
   .substring(paddedPenceNumberString.length - 2)
   .padEnd(2, "0");
 
 console.log(`£${pounds}.${pence}`);
+//This line extracts the last two digits from the padded pence number string, which represent the pence, and ensures it has exactly two digits.
```

Comment:
Extra question: Would the code still work if we delete `.padEnd(2 "0")` at line 16? Why?

Reply:
3.99 with Line 16 
3.399 without line 16

Reply:
When you test the code without `.padEnd(2,"0")`, did you run all the code or just part of it? I am not getting 3.399 on my computer.